scripts/_helpers.py

```diff
@@ -965,105 +965,94 @@ def annuity(n, r):
         return 1 / n
 
 
+# Single source for the currency reference year (aligned with `technology-data` output files / PyPSA-Earth input cost files).
+# Change this value to update the reference year everywhere.
+reference_year = 2020
+
 # Simple cache to avoid repeated computations and logging for same (currency, output_currency, year)
 _currency_conversion_cache = {}
 
 
-# Simple cache to avoid repeated computations and logging for same (currency, output_currency, year)
 def get_yearly_currency_exchange_rate(
     initial_currency: str,
     output_currency: str,
-    year: int,
     default_exchange_rate: float = None,
     _currency_conversion_cache: dict = None,
-    future_exchange_rate_strategy: str = "latest",
+    future_exchange_rate_strategy: str = "reference",  # "reference", "latest", "custom"
     custom_future_exchange_rate: float = None,
```

Comment:
Although it is a global variable of the helper, if reference_year is used we can add it as argument to the function to keep the dependency.
For example, but feel free to revise:
```suggestion
    custom_future_exchange_rate: float = None,
    reference_year: int = TECH_DATA_REFERENCE_YEAR,
```

I've seen reference_year used also in the other function below, if yes; it can be revised similarly